### PR TITLE
fix(backend): ensure a default value for Cache-Control header

### PR DESCRIPTION
**Pull request by grignards**
For all API responses.

See #5334.

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [ ] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: backend/tracim_backend/__init__.py [hunk 1]
```diff
@@ -234,6 +236,9 @@ def web(global_config: OrderedDict, **local_settings) -> Router:
     configurator.include(add_cors_support)
     # make sure to add this before other routes to intercept OPTIONS
     configurator.add_cors_preflight_handler()
+    # Ensure a "Cache-Control: no-store" is setup by default on all responses
+    # Avoid a bug with Firefox: https://github.com/tracim/tracim/issues/5334
+    configurator.add_subscriber(default_to_cache_control_no_store, NewResponse)
```
Comment (inkhey) [inline backend/tracim_backend/__init__.py:241]: This apply to error case too ? (just curious, not clear idea about caching and error case).

**Review by inkhey — CHANGES_REQUESTED** [review]
Just add a simple non-regression test for control-cache header (at least for one simple api endpoint like /system/about).

File: backend/tracim_backend/__init__.py (hunk 1 above)
Comment (grignards) [reply backend/tracim_backend/__init__.py:241]: Error case? AFAIK the htpp caching does not care about the http status code. And here the cache-control header will be added to all response as long as they do not already contain one.

Comment (inkhey) [reply backend/tracim_backend/__init__.py:241]: ok.

**Comment by Haskaris** [comment]
Tested on banch.

**Pull request merged**